app/routes/events/view/export.js

```diff
@@ -1,9 +1,20 @@
 import Ember from 'ember';
 
-const { Route } = Ember;
+const { Route, RSVP, Object } = Ember;
 
 export default Route.extend({
   titleToken() {
     return this.l10n.t('Export');
+  },
+  model() {
+    return RSVP.hash({
+      exportData: Object.create({
```

Comment:
So remove the model declaration. The variables can just exist at the controller level. You don't even have to define them anywhere.